tickets/tests.py

```diff
@@ -215,3 +218,75 @@ def setUp(self):
     def _perform_action(self, action):
         return self.client.post(reverse('tickets-ticket', args=[self.ticket.key]), {
             'ss-action': action})
+
+
+class TicketTestsIsExplicitFlagFromQueue(TicketTests):
+    """Test that the is_explicit flag of moderated sounds changes in accordance to moderator's choices"""
+
+    def setUp(self):
+        TicketTests.setUp(self)
+        self.ticket = self._create_assigned_ticket()
+
+    def _perform_action(self, action, is_explicit_flag_key=IS_EXPLICIT_KEEP_USER_PREFERENCE_KEY):
```

Comment:
if the `is_explicit_flag_key` parameter is always set, I don't think there's a reason to set a default here